eng/config/PublishData.json

```diff
@@ -141,7 +141,7 @@
       ],
       "vsBranch": "main",
       "vsMajorVersion": 17,
-      "insertionCreateDraftPR": true,
+      "insertionCreateDraftPR": false,
```

Comment:
Was this accidentally changed or is this a workflow change for us? Obviously do what works for you, just curious if we changing flow here.

Reply:
this is normal - I think we just missed flipping it back from draft when VS main switched to 17.4p2